src/EnergyPlus/Psychrometrics.cc

```diff
@@ -780,21 +787,21 @@ namespace Psychrometrics {
 
         // If below -100C,set value of Pressure corresponding to Saturation Temperature of -100C.
         if (Tkel < 173.15) {
-            Pascal = 0.0017;
+            Pascal = 0.001405102123874164;
```

Comment:
@rraustad and I noted that the value below -100°C does not match the evaluation of the equation at -100°C. So align.